pyccel/ast/numpy_wrapper.py

```diff
@@ -84,25 +86,19 @@ def get_numpy_max_acceptable_version_file():
                 body      = [],
                 results   = [Variable(name = 'b', dtype = NativeBool())])
 
-# https://numpy.org/devdocs/reference/c-api/array.html#c.PyArray_DIMS
-numpy_get_ndims = FunctionDef(name      = 'PyArray_NDIM',
+# Return the shape in the n dimension : function definition in pyccel/stdlib/cwrapper.c
```

Comment:
the function definition located at `pyccel/stdlin/cwrapper/cwrapper_ndarrays.c`

Reply:
Good catch, nothing in this file should be located in `pyccel/stdlib/cwrapper.c`

Reply:
Fixed de6219e